packages/lesswrong/server/collections/bookmarks/queries.ts

```diff
@@ -0,0 +1,50 @@
+import schema from "@/lib/collections/bookmarks/newSchema";
+import { getAllGraphQLFields } from "@/server/vulcan-lib/apollo-server/graphqlTemplates";
+import { getFieldGqlResolvers } from "@/server/vulcan-lib/apollo-server/helpers";
+import { getDefaultResolvers } from "@/server/resolvers/defaultResolvers";
+import gql from "graphql-tag";
+
+export const graphqlBookmarkQueryTypeDefs = gql`
+  type Bookmark {
+    ${getAllGraphQLFields(schema)}
+  }
+
+  # Custom selector for bookmarks
+  input BookmarkSelectorInput {
```

Comment:
Why does this need a custom selector for bookmarks?  This wouldn't be compatible with what's coming out of `getDefaultResolvers`.  (In fact I don't even think bookmarks need a `single` api interface, since it's not used in this PR, though I don't know that's it's worth excluding it for the sake of a slightly cleaner API surface right now.  But it definitely doesn't need the custom single input selector graphql type.)